packages/editor/src/block-loader.ts

```diff
@@ -12,4 +13,5 @@ export const BlockSchema = {
   'affine:page': PageBlockModel,
   'affine:list': ListBlockModel,
   'affine:group': GroupBlockModel,
+  'affine:divider': DividerBlockModel,
```

Comment:
Please also add schema definition here https://github.com/toeverything/blocksuite/blob/master/packages/store/src/__tests__/space.spec.ts#L18-L23

This is used for unit testing.

Reply:
Ok, I'll gradually add some missing code.